### ISSUE TITLE
Timeseries: Difference percent is off by factor 100

### ISSUE THREAD

**Issue by achim-k** [grafana/grafana#89879]
### What happened?

The `Difference percent` value shown in the time series legend if off by factor 100. 

![Untitled](https://github.com/grafana/grafana/assets/9250155/40791046-7800-4e89-8bbe-d08ab002cfde)
In the above screenshot, the first CPU value is 0.66 and the last one is 0.477 resulting in a difference of 27.7%, not 0.277%

### What did you expect to happen?

Correct percentage being shown

### Did this work before?

unclear, first time grafana user

### How do we reproduce it?

1. Use the demo dataset: `Demo: Graphite -> Graphite Carbon Metrics`
2. Change the panel to a time series panel
3. Add the `Difference percent` value in the legend settings
4. Observe that the shown percentage doesn't match reality

### Environment (with versions)?

Grafana:
OS:
Browser:

### Grafana platform?

I use Grafana Cloud

### Datasource(s)?

Graphite

**Comment by Sjefke61** [comment]
I can confirm this bug, difference percent should be 4.36%, not 0.0436%

![bitcoin](https://github.com/user-attachments/assets/89d69ecd-6281-4ba1-8297-6426e0c83e1d)

**Comment by achim-k** [comment]
This has regressed again, probably due to https://github.com/grafana/grafana/pull/91563. Can this issue be reopened?

**Comment by Develer** [comment]
Yes, this caused regression in a couple of places, so it's better to address it on a different level. Yes, I'll reopen it.
